api/go.mod

```diff
@@ -1,8 +1,6 @@
 module cosmossdk.io/api
 
-go 1.23.0
-
-toolchain go1.24.5
+go 1.24.6
```

Comment:
we needed to bump go cause core did, this broke a few things (the error formatting)